### PR TITLE
EWS Reports Edits

### PR DESCRIPTION
**Pull request by agrzywinski**
http://manage.dimagi.com/default.asp?162926
http://manage.dimagi.com/default.asp?162839
http://manage.dimagi.com/default.asp?162952

### CODE REVIEW

File: custom/ewsghana/reports/specific_reports/stock_status_report.py [hunk 1]
```diff
@@ -200,7 +201,9 @@ def rows(self):
             for product in products:
                 rows[product.code] = []
 
-            for d in get_second_week(self.config['startdate'], self.config['enddate']):
+            enddate = self.config['enddate']
+            startdate =self.config['startdate'] if 'custom_date' in self.config else enddate - timedelta(days=90)
```
Comment (dimagimon) [inline custom/ewsghana/reports/specific_reports/stock_status_report.py]: E225 missing whitespace around operator

File: custom/ewsghana/views.py [hunk 2]
```diff
@@ -153,6 +155,22 @@ def inventory_management(request, domain):
         )
     )
     return HttpResponse(
-        json.dumps(inventory_management_ds.charts[0].data),
+        json.dumps(inventory_management_ds.charts[0].data, default=json_handler),
+        mimetype='application/json'
+    )
+
+@require_GET
```
Comment (dimagimon) [inline custom/ewsghana/views.py]: E302 expected 2 blank lines, found 1

File: custom/ewsghana/templates/ewsghana/multi_report.html [hunk 3]
```diff
@@ -4,6 +4,9 @@
 {% load i18n %}
 
 {% block reportcontent %}
+    {% if report.slug|safe == 'stock_summary_report' or report.slug|safe == 'cms_rms_summary_report' %}
```
Comment (czue) [inline custom/ewsghana/templates/ewsghana/multi_report.html]: feels like this type of hard-coding could be a bit brittle. maybe you could make an attribute on the report class like `show_title` or something and pass it to the template?

File: custom/ewsghana/templates/ewsghana/multi_report.html [hunk 4]
```diff
@@ -153,6 +162,16 @@ <h4 class="media-heading">{{ thisreport.report_table.title }}<br/><small>{{ subt
         {% else %}
             <span class="report">
                 <h4 class="media-heading">{{ thisreport.report_table.title }}<br/><small>{{ subtitle1 }}</small><br/><small>{{ subtitle2 }}</small></h4>
+                {% if thisreport.report_table.slug == 'stockouts_product' %}
```
Comment (czue) [inline custom/ewsghana/templates/ewsghana/multi_report.html]: same comment here

**Comment by czue** [comment]
don't think the comments need to block merge but could be good to follow up

**Pull request merged**